### ISSUE TITLE
Not all time zones receive push notifications

### ISSUE THREAD

**Issue by klaaspieter** [thoughtbot/Tropos#185]
Tropos broadcasts notifications to a Courier channel using the time zone name. Unfortunately the canonical names of the time zones don't always match. For example setting an iPhone to use San Francisco time zone returns “US/Pacific”. Tropos will thus [register with channel](https://github.com/thoughtbot/Tropos/blob/kp-release/Tropos/Sources/Controllers/TRApplicationController.m#L63) `us-pacific`. 

The backend uses [`ActiveSupport::Timezone`](http://api.rubyonrails.org/classes/ActiveSupport/TimeZone.html) to determine in which time zones it's currently 9am and broadcast to those time zones. Today at 9am pacific time the backend returns: `["America/Los_Angeles", "America/Tijuana", "America/Phoenix"]` which does not include `us-pacific`.

In other words iOS is subscribing to different channels than the backend is broadcasting to. It's possible for users in these time zones to never receive notifications.

**Comment by klaaspieter** [comment]
This was resolved by #187.
